### PR TITLE
Match network configuration with Chainlist

### PR DESCRIPTION
**Pull request by zkokelj**
### Why this change is needed

Currently MetaMask shows multiple warnings because network added does not match networks that are supported.
We want to add networks without unnecessary warnings.

### What changes were made as part of this PR

Changed information about new network to match: https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-443.json

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

**Comment by tudor-malene** [comment]
@zkokelj , is this blocked by something?

**Review by tudor-malene — APPROVED** [review]
lgtm
(@otherview , do you think there's anything missing?)

**Comment by BedrockSquirrel** [comment]
LGTM but if you do a find across the codebase for '777' I think there's a couple more worth replacing (e.g. configure-metamask.md and some mentions of "default: 777" in script comments)

**Pull request merged**
